metadata/site_import/services.xml

```diff
@@ -1,33 +1,43 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <services xmlns="http://www.demandware.com/xml/impex/services/2014-09-26">
-     <service-credential service-credential-id="AdyenPayment">
+    <service-credential service-credential-id="AdyenPayment">
         <url>https://checkout-test.adyen.com/checkout/v41/payments</url>
-        <user-id>yourWebServiceUser</user-id>
+        <user-id></user-id>
         <password/>
     </service-credential>
-    <service-credential service-credential-id="AdyenPayment3DSecure">
-        <url>https://checkout-test.adyen.com/checkout/v41/payments/details</url>
-        <user-id>yourWebServiceUser</user-id>
+    <service-credential service-credential-id="AdyenPaymentLive">
+        <url>https://[YOUR_LIVE_PREFIX]-checkout-live.adyen.com/checkout/v41/payments</url>
```

Comment:
Do merchants still need to update the service urls with their live prefix? Can't this be done from the Custom Preferences section?

Reply:
No, this will not be done through Custom Preferences. Deleted the config there as well.